chainbase/src/main/java/org/tron/core/service/RewardViCalService.java

```diff
@@ -55,10 +53,11 @@ public class RewardViCalService {
 
   private volatile long lastBlockNumber = -1;
 
-  @VisibleForTesting
-  @Setter
-  private Sha256Hash rewardViRoot = Sha256Hash.wrap(
-      ByteString.fromHex("9debcb9924055500aaae98cdee10501c5c39d4daa75800a996f4bdda73dbccd8"));
+  private static final String MAIN_NET_ROOT_HEX =
+      "9debcb9924055500aaae98cdee10501c5c39d4daa75800a996f4bdda73dbccd8";
```

Comment:
this root hex is already configured in conf. here it is declared again. Should it be possible for this value declared in one place instead of two different files？

Reply:
This is to ensure that the main-net node does not need to change the configuration file.